### ISSUE TITLE
Dicom files under import/SampleStudy2 cannot be displayed in Cornerstone3D

### ISSUE THREAD

**Issue by xuwenhao2525** [knopkem/dicomweb-archive#25]
it does not seem to support Cornerstone3D because images cannot be viewed in Cornerstone3D，I found that sample1 and sample2 under the directory import/SampleStudy1 can be displayed in Cornerstone3D, but dicom files under the directory import/SampleStudy2 cannot be displayed in Cornerstone3D. Why is this? We look forward to your reply!

**Comment by knopkem** [comment]
Images need properties that define their position and orientation in space (Dicom Attribues: Image Orientation Patient and Image Position Patient). For images within on series those positions and orientation should also be related (some coordinate system, same orientation, distance between consecutive slices always the same etc.), see https://dicomiseasy.blogspot.com/2013/06/getting-oriented-using-image-plane.html
So tl;dr: It's not a bug, you won't find any viewer that can render those images in 3D
